app/tests/components/Form/ApplicationForm.test.tsx

```diff
@@ -215,38 +215,45 @@ describe('The application form', () => {
       query: data.query,
     }));
 
-    const continueButton = screen.getByRole('button', {
-      name: 'Save and continue',
-    });
-    //node here is using the jest expect, whereas TS can only find the cypress jest
-    expect(continueButton.hasAttribute('disabled')).toBeTrue();
+    expect(
+      screen.getByRole('button', {
+        name: 'Save and continue',
+      })
+    ).toBeDisabled();
   });
 
   it('acknowledgement page continue is enabled once all checkboxes have been clicked', async () => {
-    componentTestingHelper.loadQuery();
+    componentTestingHelper.loadQuery({
+      ...mockQueryPayload,
+      Application() {
+        return {
+          id: 'TestApplicationID',
+          formData: {
+            ...mockFormData,
+            acknowledgements: {
+              acknowledgementsList: acknowledgementsEnum.slice(0, -1),
```

Comment:
This almost feels like cheating but it does serve the same purpose. Do we have any care about testing for clicking each box or since we have it covered in our e2e tests it's not a concern?

Reply:
reverted the changes